src/java/org/apache/cassandra/service/accord/AccordMessageSink.java

```diff
@@ -222,7 +222,7 @@ public void send(Node.Id to, Request request)
         Preconditions.checkNotNull(verb, "Verb is null for type %s", request.type());
         Message<Request> message = Message.out(verb, request);
         InetAddressAndPort endpoint = endpointMapper.mappedEndpoint(to);
-        logger.debug("Sending {} {} to {}", verb, message.payload, endpoint);
```

Comment:
this is super dense / verbose and slows things down and makes real bugs hard to find in the logs...